pkgs/development/rocm-modules/6/hipblas/default.nix

```diff
@@ -63,13 +85,16 @@ stdenv.mkDerivation (finalAttrs: {
 
   cmakeFlags =
     [
-      "-DCMAKE_C_COMPILER=hipcc"
-      "-DCMAKE_CXX_COMPILER=hipcc"
+      "-DCMAKE_BUILD_TYPE=Release"
+      "-DCMAKE_CXX_COMPILER=${lib.getExe' clr "hipcc"}"
+      # Upstream is migrating to amdclang++, it is likely this will be correct in next version bump
+      #"-DCMAKE_CXX_COMPILER=${lib.getBin clr}/bin/amdclang++"
       # Manually define CMAKE_INSTALL_<DIR>
       # See: https://github.com/NixOS/nixpkgs/pull/197838
       "-DCMAKE_INSTALL_BINDIR=bin"
       "-DCMAKE_INSTALL_LIBDIR=lib"
       "-DCMAKE_INSTALL_INCLUDEDIR=include"
+      "-DAMDGPU_TARGETS=${rocblas.amdgpu_targets}" # FIXME:
```

Comment:
Was there a comment supposed to go after the FIXME?

Reply:
Oops